website/docs/search.md

```diff
@@ -37,22 +37,19 @@ By default, the Docusaurus preset generates a [sitemap.xml](https://docusaurus.i
 
 :::
 
-### Index Configuration {#algolia-index-configuration}
+:::info From an old docsearch version?
 
-After applying, your site's DocSearch config should be created at:
+You can read more about migration from DocSearch v2 and configurating v3 in [our blog post](/blog/2021/11/21/algolia-docsearch-migration).
 
-```
-https://github.com/algolia/docsearch-configs/blob/master/configs/<indexName>.json
-```
+:::
 
-This configuration file can be updated by:
+### Index Configuration {#algolia-index-configuration}
 
-- [**asking for help**](#algolia-support): the DocSearch team can help you maintain it
-- opening a pull-requests in [algolia/docsearch-configs](https://github.com/algolia/docsearch-configs)
+After your application has been approved, you can view and update your search configuration through the web interface.
```

Comment:
```suggestion
After your application has been approved and deployed, you will receive an email with all the details for you to add DocSearch to your project. Editing and managing your crawls can be done via [the web interface](https://crawler.algolia.com/)
```

It's maybe worth mentioning that indices are ready to use when deployed, which means configuration is not necessary, wdyt?